README.md

```diff
@@ -1,70 +1,65 @@
+cd laa-crime-assessment-service
+```
 
-### Read about the GitHub repository standards
+### Java set-up
 
-Familiarise yourself with the Ministry of Justice GitHub Repository Standards. These standards ensure consistency, maintainability, and best practices across all our repositories.
+If you do not already have Java 21 installed, you will need to install it. There are many ways to do this, but a recommended approach
+is to use [jenv](https://github.com/jenv/jenv) to manage your local Java versions and enable easy switching between them. jenv will
+also automatically switch to use the correct Java version when you navigate to a project containing a `.java-version` file (providing
+it is already installed).
 
-You can find the standards [here](https://github-community.service.justice.gov.uk/repository-standards/guidance).
+```sh
+# Check your current Java version - if it is Java 21 then the rest of the following steps are optional
+java -version
 
-Please read and understand these standards thoroughly and enable them when you feel comfortable.
+brew update
 
-### Modify the GitHub Standards Badge
+brew install jenv
+brew install openjdk@21
 
-Once you've ensured that all the [GitHub Repository Standards](https://github-community.service.justice.gov.uk/repository-standards/guidance) have been applied to your repository, it's time to update the Ministry of Justice (MoJ) Compliance Badge located in the README file.
+# See the output from the previous command to determine where Java 21 was installed
+# You may need to symlink to your JavaVirtualMachines directory from the installed location to be able to use it
+jenv add /path/to/Java/Contents/Home
 
-The badge demonstrates that your repository is compliant with MoJ's standards.
+# Check to ensure that you can see Java 21 now listed
+jenv versions
 
-To update the badge, replace the `template-repository` in the badge URL with your repository's name. The badge URL should look like this:
+# Switch to Java 21 (ensure you're still inside the laa-crime-assessment-service directory)
+jenv local 21
 
-```markdown
-[![Ministry of Justice Repository Compliance Badge](https://github-community.service.justice.gov.uk/repository-standards/api/${your-repository-name}/badge)](https://github-community.service.justice.gov.uk/repository-standards/${your-reposistory-name})
+# Confirm that Java 21 is now the current Java version
+java -version
 ```
 
-**Please note** the badge will not function correctly if your repository is internal or private. In this case, you may remove the badge from your README.
-
-### Update CODEOWNERS
+### Running tests
 
-(Optional) Modify the CODEOWNERS file to specify the teams or users authorized to approve pull requests.
+Make sure tests all testes are passed by running following ‘gradle’ Command
```

Comment:
Sentence is a little wonky, I assume you meant something like **Make sure all tests are passing by running the following `gradle` command:**?

Reply:
Thanks Nigel, I copied this bit from another README so the sentence is wonky in a few places. Will update now!

Reply:
Done